tests/vmi_hook_sidecar_test.go

```diff
@@ -293,7 +293,7 @@ var _ = Describe("[sig-compute]HookSidecars", decorators.SigCompute, func() {
 
 		Context("with ConfigMap in sidecar hook annotation", func() {
 
-			DescribeTable("should update domain XML with SM BIOS properties", func(withImage bool) {
+			It("[test_id:10907] should update domain XML with SM BIOS properties", func(withImage bool) {
```

Comment:
`DescribeTable` is used here to run a table driven set of tests.
You cannot simply mark it as a scalar node with `It`.

Reply:
fixed now
I just added the polarionID to DescribeTable